### PR TITLE
[Hexagon] disable cache_write schedule type from sw pipeline test

### PR DESCRIPTION
**Pull request by adstraw**
Disabling the "cache_write" schedule from this test as it is failing on device.  The root cause of the failure will be addressed with additional buffer dependency analysis in the InjectSoftwarePipeline pass to come in a subsequent PR.

### CODE REVIEW

**Comment by masahi** [comment]
I'm seeing "1 workflow awaiting approval (First-time contributors need a maintainer to approve running workflows)" in this PR, while @adstraw is obviously not a first time contributor. Why is that? cc @driazati

**Comment by driazati** [comment]
> I'm seeing "1 workflow awaiting approval (First-time contributors need a maintainer to approve running workflows)" in this PR, while @adstraw is obviously not a first time contributor. Why is that? cc @driazati
> 
> UPDATE: #13005 has the same issue.

Not sure why (seems like a bug from GitHub?), since even on the hover on @adstraw's profile it says there are recent contributions. ASF Infra might have changed something in the repo settings but it seems unlikely that they would do that without telling us / it would even cause this kind of behavior. I've approved both PRs, if you notice anything else like this please tag me on it

**Review by tmoreau89 — APPROVED** [review]
LGTM

**Pull request merged**
